src/js/download_script.js

```diff
@@ -13,68 +13,58 @@
 if (typeof plxDwnld === "undefined") {
 
     window.plxDwnld = (function () {
-
         const self = {};
-        const clientIdRegex = new RegExp("server\/([a-f0-9]{40})\/");
-        const metadataIdRegex = new RegExp("key=%2Flibrary%2Fmetadata%2F(\\d+)");
-        const apiResourceUrl = "https://plex.tv/api/resources?includeHttps=1&X-Plex-Token={token}";
-        const apiLibraryUrl = "{baseuri}/library/metadata/{id}?X-Plex-Token={token}";
-        const downloadUrl = "{baseuri}{partkey}?X-Plex-Token={token}";
-        const accessTokenXpath = "//Device[@clientIdentifier='{clientid}']/@accessToken";
-        const baseUriXpath = "//Device[@clientIdentifier='{clientid}']/Connection[@local='0']/@uri";
-        const partKeyXpath = "//Media/Part[1]/@key";
-        let accessToken = null;
         let baseUri = null;
 
         const getXml = function (url, callback) {
             const request = new XMLHttpRequest();
-            request.onreadystatechange = function () {
+            request.onload = () => {
                 if (request.readyState == 4 && request.status == 200) {
                     callback(request.responseXML);
+                } else {
+                    alert("Unsuccessful XML URL: " + url);
```

Comment:
I'd rather make these `console.error()`s at most; better to fail silently than be disruptive.